nevergrad/optimization/optimizerlib.py

```diff
@@ -2615,7 +2653,20 @@ def _select_optimizer_cls(self) -> base.OptCls:
 
 
 @registry.register
-class NGOpt(NGOpt14):
+class NGOpt15(NGOpt14):  # Also known as NGOpt12H_nohyperopt
+    def _select_optimizer_cls(self) -> base.OptCls:
+        if (
+            self.budget is not None
+            and self.budget < self.dimension**2 * 2
+            and self.num_workers == 1
+        ):
+            return OnePlusOne  # MetamodelOpO seems equivalent so far
+        else:
+            return super()._select_optimizer_cls()
```

Comment:
This is an enormous change as this impact NGOpt. This changes the main optimization method in Nevergrad. This can not be merged without very carefully checking that it's better than the previous NGOpt.

Reply:
Additional thought: it makes NGOpt sequential, which could be unexpected

Reply:
It chooses 1+1 only if n_workers = 1. Isn't it already the case of a sequential opt strategy?